specification/communication/data-plane/Common/stable/2022-07-13/common.json

```diff
@@ -84,6 +84,30 @@
         "modelAsString": true
       }
     },
+    "CallingCommunicationIdentifierModel": {
```

Comment:
Do we need the Model suffix?

Reply:
I'll note this does seem to match the existing convention used in CommunicationIdentifierModel, CommunicationUserIdentifierModel, etc.

Reply:
This is missing the "type" property.